packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -268,6 +258,18 @@ class _FloatingActionButtonState extends State<FloatingActionButton> {
       child: result,
     );
 
+    if (widget.tooltip != null) {
+      final Widget tooltip = MergeSemantics(
```

Comment:
I'm not sure if we actually want a MergeSemantics here, or if we just want to hide the tooltip label entirely using `excludeFromSemantics`. cc @goderbauer

Reply:
This will cause the FAB to take on the tooltip label, where previously it did not

Reply:
Ahh okay cool, that makes more sense then our behavior

Reply:
Yes, I used MergeSemantics to keep the semantics the same as before.